packages/workflow/src/records/fhir.ts

```diff
@@ -613,6 +614,9 @@ export function createCorrectedTask(
       {
         url: MAKE_CORRECTION_EXTENSION_URL,
         valueString: 'REGISTERED'
+      },
```

Comment:
Does the correction + unassignment show correctly in record audit if it's both in the same task? I see there's a function `createUnassignedTask` (Which should use the same constant used here so it's easier to find) that at least creates a completely separate task

Reply:
It shows `Unassigned` in record audit not `Record corrected`. Should there be two tasks, `Unassigned` and `Record corrected` ?

Reply:
Or we could change the order of search in [`getActionFromTask()`](https://github.com/opencrvs/opencrvs-core/blob/d1980ed2179ab743bd2d6bce5b96c67797ea2304/packages/commons/src/fhir/task.ts#L305) to check if there exists `MAKE_CORRECTION_EXTENSION_URL` before checking `UNASSIGNED_EXTENSION_URL`

Reply:
There should be two tasks 👍